lua/damagelogs/shared/autoslay.lua

```diff
@@ -119,6 +132,18 @@ local function CreateCommand()
 end
 hook.Add("Initialize", "AutoSlay", CreateCommand)
 
+if SERVER then
+	hook.Add( "TTTEndRound", "AutoSlay_CheckForMarked", function( res )
+		if not #Damagelog.markedForSlay > 0 then return end -- No need to do anything if no one is marked to be slain
+		for i = 1, #Damagelog.markedForSlay do
+			local data = Damagelog.markedForSlay[ i ]
+			Damagelog:AddSlays( data.admin, data.steamid, data.rounds, data.reason, data.target )
```

Comment:
AddSlays seems not to be part of the original repo.

Reply:
[Here](https://github.com/Tommy228/TTTDamagelogs/blob/master/lua/damagelogs/server/autoslay.lua#L117)

Reply:
Yes **Set**Slays, but you are using **Add**Slays. 😛